fabric/src/main/java/io/github/gaming32/bingo/fabric/datagen/goal/EasyGoalProvider.java

```diff
@@ -348,7 +348,7 @@ public void addGoals() {
         // TODO: different colored shields
         addGoal(obtainItemGoal(id("dead_bush"), Items.DEAD_BUSH)
             .tags(BingoTags.OVERWORLD, BingoTags.RARE_BIOME));
-        addGoal(obtainItemGoal(id("grass"), Items.GRASS, 15, 32)
+        addGoal(obtainItemGoal(id("grass"), Items.SHORT_GRASS, 15, 32) // FIXME: Support TALL_GRASS too
```

Comment:
This can easily be done with an item tag

Reply:
I felt like this FIXME was something to be done immediately after the merge, as it's not really related to the version bump.